x-pack/plugins/data_visualizer/public/application/index_data_visualizer/embeddables/field_stats/field_stats_factory.tsx

```diff
@@ -0,0 +1,390 @@
+          // PublishesDataLoading
+          dataLoading,
+          // PublishesBlockingError
+          blockingError,
+          getTypeDisplayName: () =>
+            i18n.translate('xpack.dataVisualizer.fieldStats.typeDisplayName', {
+              defaultMessage: 'field statistics',
+            }),
+          isEditingEnabled: () => true,
+          onEdit: async () => {
+            try {
+              const { resolveEmbeddableFieldStatsUserInput } = await import(
+                './resolve_field_stats_embeddable_input'
+              );
+              const chartState = serializeFieldStatsChartState();
+              const nextUpdate = await resolveEmbeddableFieldStatsUserInput(
+                coreStart,
+                pluginStart,
+                parentApi,
+                uuid,
+                false,
+                chartState,
+                fieldStatsControlsApi
+              );
+
+              fieldStatsControlsApi.updateUserInput(nextUpdate);
+            } catch (e) {
+              return Promise.reject(e);
+            }
+          },
+          dataViews: dataViews$,
+          serializeState: () => {
+            const dataViewId = fieldStatsControlsApi.dataViewId$?.getValue();
+            const references: Reference[] = dataViewId
+              ? [
+                  {
+                    type: DATA_VIEW_SAVED_OBJECT_TYPE,
+                    name: FIELD_STATS_DATA_VIEW_REF_NAME,
+                    id: dataViewId,
+                  },
+                ]
+              : [];
+            return {
+              rawState: {
+                ...serializeTitles(),
+                ...serializeTimeRange(),
+                ...serializeFieldStatsChartState(),
+              },
+              references,
+            };
+          },
+        },
+        {
+          ...timeRangeComparators,
+          ...titleComparators,
+          ...fieldStatsControlsComparators,
+        }
+      );
+
+      const reload$ = fetch$(api).pipe(
+        skipWhile((fetchContext) => !fetchContext.isReload),
+        map(() => Date.now())
+      );
+      const reset$ = resetData$.pipe(skip(1), distinctUntilChanged());
+
+      const onTableUpdate = (changes: Partial<DataVisualizerTableState>) => {
+        if (isDefined(changes?.showDistributions)) {
+          fieldStatsControlsApi.showDistributions$.next(changes.showDistributions);
+        }
+      };
+
+      const addFilters = (filters: Filter[], actionId: string = ACTION_GLOBAL_APPLY_FILTER) => {
+        if (!pluginStart.uiActions) {
+          toasts.addWarning(ERROR_MSG.APPLY_FILTER_ERR);
+          return;
+        }
+        const trigger = pluginStart.uiActions.getTrigger(APPLY_FILTER_TRIGGER);
+        if (!trigger) {
+          toasts.addWarning(ERROR_MSG.APPLY_FILTER_ERR);
+          return;
+          // eslint-disable-next-line no-console
+          console.error(`${ERROR_MSG.APPLY_FILTER_ERR}: APPLY_FILTER_TRIGGER is undefined`);
```

Comment:
console.error can be removed. Its after a return so this code will never get run

Reply:
Updated here https://github.com/elastic/kibana/pull/184030/commits/1cad8784ad2ecc05fa10040827a7a2fc0d434c7b